test/test-repo/datastore/LOG.old

```diff
@@ -0,0 +1,7 @@
+=============== Mar 16, 2017 (CET) ===============
+23:13:26.858744 log@legend F·NumFile S·FileSize N·Entry C·BadEntry B·BadBlock Ke·KeyError D·DroppedEntry L·Level Q·SeqNum T·TimeElapsed
+23:13:26.859139 db@open opening
+23:13:26.859272 journal@recovery F·1
+23:13:26.859951 journal@recovery recovering @1
+23:13:26.862700 db@janitor F·2 G·0
+23:13:26.862790 db@open done T·3.618327ms
```

Comment:
Include these files in .gitignore, otherwise they will be getting updated constantly.